packages/apputils/test/inputdialog.spec.ts

```diff
@@ -144,6 +144,48 @@ describe('@jupyterlab/apputils', () => {
       });
     });
 
+    describe('getMultipleItems()', () => {
+      it('should accept at least two arguments', async () => {
+        const dialog = InputDialog.getMultipleItems({
+          title: 'list',
+          items: ['item1']
+        });
+
+        await dismissDialog();
+        expect((await dialog).button.accept).toBe(false);
+      });
+
+      it('should return empty list if none is selected', async () => {
+        const dialog = InputDialog.getMultipleItems({
+          items: ['item1', 'item2'],
+          title: 'Pick a choice'
+        });
+
+        await acceptDialog();
+
+        const result = await dialog;
+
+        expect(result.button.accept).toBe(true);
+        expect(result.value).toStrictEqual([]);
+      });
+
+      it('should accept option "currents"', async () => {
```

Comment:
The test description does not convey the test goal. I guess you mean `defaults`:

```suggestion
      it('should accept option "defaults"', async () => {
```